HTML/Vehicles.html

```diff
@@ -1,8 +1,128 @@
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+    </div>
+    <div class="carRow">
+        <div class="carListing 1">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 2">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 3">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 4">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+    </div>
+    <div class="carRow">
+        <div class="carListing 1">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 2">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 3">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 4">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+    </div>
+    <div class="carRow">
+        <div class="carListing 1">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 2">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 3">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 4">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+    </div>

```

Comment:
The structure of the car rows is perfect!

Reply:
thanks